### PR TITLE
Monitoring your microservices with Micrometer.io

### PR DESCRIPTION
**Pull request by TYsewyn**

### CODE REVIEW

File: _posts/2017-09-15-monitoring-your-microservice-with-micrometer.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+A meter is uniquely identified by its name and dimensions (also called tags).
+Dimensions are a way of adding dimensions to metrics, so they can be sliced, diced, aggregated and compared.
+For example, we have a meter named `http.requests` with a tag `uri`.
+With this meter we could see the overall amount of HTTP request, but also have the option to drill down and see the amount of HTTP requests for a specific URI.
+
+### Counters
+
+Counters are a cumulative metric that represents a single numerical value that only ever goes up.
+They are typically used to count requests served, tasks completed, errors occurred, etc.
+Counters should not be used to expose current counts of items whose number can also go down, gauges are a better fit for this use case.
```
Comment (Turbots) [inline _posts/2017-09-15-monitoring-your-microservice-with-micrometer.md]: make this last line bold? seems important

File: _posts/2017-09-15-monitoring-your-microservice-with-micrometer.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Quantiles are cutpoints dividing the range of a probability distribution into contiguous intervals with equal probabilities, or dividing the observations in a sample in the same way.
+Timers and distribution summaries can be enriched with quantiles computed in your app prior to shipping to a monitoring backend.
+Depending on the size of your deployments, computing quantiles at instrumentation time may or may not be useful.
+It is **not possible** to aggregate quantiles across a cluster.
+
+Four quantile algorithms are provided out of the box with different tradeoffs:
+
+- `WindowSketchQuantiles` - The importance of an observation is decayed as it ages. This is the most computationally costly algorithm.
+- `Frugal2UQuantiles` - Successive approximation algorithm that converges towards the true quantile with enough observations. This is by least costly algorithm, but exhibits a higher error ratio in early observations.
```
Comment (Turbots) [inline _posts/2017-09-15-monitoring-your-microservice-with-micrometer.md]: This is **the** least costly algorithm (typo)

File: _posts/2017-09-15-monitoring-your-microservice-with-micrometer.md (hunk 1 above)
Comment (TYsewyn) [reply _posts/2017-09-15-monitoring-your-microservice-with-micrometer.md]: I changed it to "Counters should **not** be used to expose current counts"

File: _posts/2017-09-17-monitoring-your-microservices-with-micrometer.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+There are a couple of ways to create meters.
+We will cover all different types, when to use them, and furthermore how to implement them.
+
+### Dimensions/Tags
+
+A meter is uniquely identified by its name and dimensions (also called tags).
+Dimensions are a way of adding dimensions to metrics, so they can be sliced, diced, aggregated and compared.
+For example, we have a meter named `http.requests` with a tag `uri`.
+With this meter we could see the overall amount of HTTP request, but also have the option to drill down and see the amount of HTTP requests for a specific URI.
```
Comment (yannickdeturck) [inline _posts/2017-09-17-monitoring-your-microservices-with-micrometer.md]: Http request**s**

**Pull request merged**